OurUmbraco/Documentation/DocumentationUpdater.cs

```diff
@@ -223,6 +223,15 @@ private SiteMapItem GetFolderStructure(DirectoryInfo dir, string rootPath, int l
                         case "frequently-asked-questions":
                             return 6;
                         
+                        //Umbraco Heartcore
+                        case "getting-started-cloud"
+                            return 0;
+                        case "api-documenation"
+                            return 1;
+                        case "client-libraries"
+                            return 2;
+                        case "versions-and-updates"
+                            return 3;
```

Comment:
Looks great @OwainWilliams !
I've made a few suggestions: looks like a colon needs to be added after each case.

```suggestion
                        case "getting-started-cloud":
                            return 0;
                        case "api-documenation":
                            return 1;
                        case "client-libraries":
                            return 2;
                        case "versions-and-updates":
                            return 3;
```